### ISSUE TITLE
Need custimization options for users

### ISSUE THREAD

**Issue by CrypticCus** [purogamer/Fluent-for-Steam#89]
I would like to remove wide screen mode from left sidebar. and add another shortcuts to there please.

**Comment by xKiisa** [comment]
Agree with the heart button, it feels out of place.

**Comment by prodarma** [comment]
Agreed.

**Comment by Tethgar** [comment]
Agreed. The heart itself wouldn't be much of an issue if it 1. were in the screenshots he uploaded or 2. included instructions in the readme on how to remove it.
